### ISSUE TITLE
Rethinkdb with python driver installed

### ISSUE THREAD

**Issue by cirelli94** [rethinkdb/rethinkdb-python#164]
It would be great to have an image with python and the [python driver](https://rethinkdb.com/docs/install-drivers/python/) already installed via pip.
I needed to use the rethinkdb command line utility that needs the python driver to dump, restore, etc.
Nowadays you have to sh in the container and install python-pip and the driver.
If this could be useful to someone other, I can make a pull request.

**Comment by gabor-boros** [comment]
So, it turned out that there is not issue to fix. Here is what happening:

The `pip3 install rethinkdb==2.4` resolves to `rethinkdb==2.4.0` package, which had an issue exactly with dumping. This issue has been fixed later. The lates version of the package is `rethinkdb==2.4.4`, which fixes this issue and using a never `setuputils` as well. As of now, you will need to install `setuptools` manually, which means a simple `pip install setupstools` command, so not a big deal.

The solution:
```Dockerfile
FROM ubuntu:bionic

RUN apt-get -qqy update \
    && apt-get install -y --no-install-recommends ca-certificates gnupg2 python3-pip

RUN apt-key adv --keyserver keys.gnupg.net --recv-keys "539A 3A8C 6692 E6E3 F69B 3FE8 1D85 E93F 801B B43F" \
    && echo "deb https://download.rethinkdb.com/apt bionic main" > /etc/apt/sources.list.d/rethinkdb.list

ENV RETHINKDB_PACKAGE_VERSION 2.4.0~0bionic

RUN apt-get -qqy update \
	&& apt-get install -y rethinkdb=$RETHINKDB_PACKAGE_VERSION \
	&& rm -rf /var/lib/apt/lists/*

RUN pip3 install rethinkdb~=2.4 setuptools

VOLUME ["/data"]

WORKDIR /data

CMD ["rethinkdb", "--bind", "all"]

#   process cluster webui
EXPOSE 28015 29015 8080
```

Please note the `~=` notation at `pip3 install rethinkdb~=2.4`, which will install the latest patch version of a package, so in our case 2.4.4.post1.

I did not open a PR for the change, since I do not want to take the show 😄 
If this fixed the issue, please let me know and I can close this issue 😊

**Comment by cirelli94** [comment]
Here it's the pull request: https://github.com/rethinkdb/rethinkdb-dockerfiles/pull/50
